seed/seed.json

```diff
@@ -2948,7 +2948,7 @@
                             "Playlist"
                         ]
                     },
-                    "name": "Enabled",
+                    "name": "Enabledd",
```

Comment:
What's this about?  Is it an intentional change, or a typo?

The rest looks good to me for desktop 👍

Reply:
it's a test change to trigger the seed generation.